src/libexpr/eval.cc

```diff
@@ -503,7 +505,17 @@ EvalState::EvalState(
     , sOutputSpecified(symbols.create("outputSpecified"))
     , repair(NoRepair)
     , emptyBindings(0)
-    , derivationInternal(rootPath(CanonPath("/builtin/derivation.nix")))
+    , rootFS(makeFSInputAccessor(CanonPath::root))
+    , corepkgsFS(makeMemoryInputAccessor())
+    , internalFS(makeMemoryInputAccessor())
```

Comment:
I don't think we need to store `internalFS`. Wouldn't it be ok to let this value only be reachable through `callFlakeInternal`? `internalFS` isn't referenced anywhere else.

Reply:
It's accesses in lazy-trees and I want to minimize the diff. Also, we may want to add more files to `internalFS` in the future.